<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlg_change_directory</class>
 <widget class="QDialog" name="dlg_change_directory">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>286</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change data directory for current  account</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label1">
     <property name="text">
      <string>Data for this account is currenly stored in:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="currentPath">
     <property name="font">
      <font>
       <italic>true</italic>
      </font>
     </property>
     <property name="text">
      <string>n/a</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label2">
     <property name="text">
      <string>Select a new directory where data should be stored:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label3">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>New data directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="newDirectoryComboBox"/>
   </item>
   <item>
    <widget class="QRadioButton" name="moveDataRadioButton">
     <property name="text">
      <string>Move data to the new directory</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="copyDataRdioButton">
     <property name="text">
      <string>Copy data to the new directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="StartDataRadioButton">
     <property name="text">
      <string>Start afresh in the new directory</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label4">
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Cannot used the original directory as destination</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlg_change_directory</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlg_change_directory</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
